README.md

```diff
@@ -1,5 +1,7 @@
-libintel\_intriniscs for Unikraft
-===================
+# Intel (x86) intrinsics for Unikraft
 
-Please refer to the `README.md` as well as the documentation in the `doc/`
-subdirectory of the main unikraft repository.
+This is a port of the headers providing compiler intrinsics for x86 architectures.
+A list of these intrinsics can be browsed at [Intel Intrinsics](https://www.intel.com/content/www/us/en/docs/intrinsics-guide/index.html).
+Normally you would receive these headers together with your compiler, however, due to Unikraft's isolated build environment, we provide them explicitly.
+
+Please refer to `README.md` in the main unikraft repository as well as the [Unikraft Homepage](https://unikraft.org/) for further information.
```

Comment:
```suggestion
Please refer to `README.md` in the main unikraft repository as well as the [Unikraft Homepage](https://unikraft.org/) for further information.
```

Please provide a link to the `README.md` file

Reply:
I don't think that's necessary @RaduNichita. Check the other libs, neither of them links the core Unikraft README which is also easily available. Consistency between repos is key.

Reply:
OK, fair point here